src/main/resources/twitter-demo/tweetbook.tpl

```diff
@@ -18,6 +18,7 @@
     <script src="static/js/dc.min.js"></script>
     <script src="static/js/tweetbook.js"></script>
     <script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>
+    <script src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
I think we should add the “http:” prefix. 
> On Apr 4, 2016, at 9:12 PM, Chen Li <notifications@github.com> wrote:
> 
> In src/main/resources/twitter-demo/tweetbook.tpl <https://github.com/ISG-ICS/cloudberry/pull/10#discussion_r58484631>:
> 
> > @@ -18,6 +18,7 @@
> >      <script src="static/js/dc.min.js"></script>
> >      <script src="static/js/tweetbook.js"></script>
> >      <script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>
> > +    <script src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
> Sorry I not sure, twitter's default api generate this format
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/ISG-ICS/cloudberry/pull/10/files/6b7c45035e89bab3da6b4c0e275a8bd2dc944d93#r58484631>



Best,

Jianfeng Jia
PhD Candidate of Computer Science
University of California, Irvine